client/datatable-bs5.js

```diff
@@ -0,0 +1,18 @@
+
+// pwi 2024- 6- 4
+//  try here to mutualize the datatable imports
+//  begin
+import 'datatables.net-buttons-bs5';
+import 'datatables.net-buttons/js/buttons.colVis.mjs';
+import 'datatables.net-buttons/js/buttons.print.mjs';
+import 'datatables.net-colreorder-bs5';
+import 'datatables.net-fixedheader-bs5';
+import 'datatables.net-responsive-bs5';
+import 'datatables.net-scroller-bs5';
+//
+import 'datatables.net-bs5/css/dataTables.bootstrap5.min.css';
+import 'datatables.net-buttons-bs5/css/buttons.bootstrap5.min.css';
+import 'datatables.net-colreorder-bs5/css/colReorder.bootstrap5.min.css';
+import 'datatables.net-fixedheader-bs5/css/fixedHeader.bootstrap5.min.css';
+import 'datatables.net-responsive-bs5/css/responsive.bootstrap5.min.css';
+import 'datatables.net-scroller-bs5/css/scroller.bootstrap5.min.css';
```

Comment:
@p-wieser can you describe me why these are now mandatory to be imported, while they have not been mandatory before? How were datatables styles imported before?

Reply:
Yes, good point. Indeed these reflect my own design choices :( So should rather be imported elsewhere..
Seems that only extension stylesheets have to be imported (https://datatables.net/download/).
I'll fix that.